src/commands/index.js

```diff
@@ -14,6 +14,8 @@ module.exports = function (client) {
   };
 
   client.appdistribution = {};
+  client.appdistribution.addTesters = loadCommand("appdistribution-testers-add");
+  client.appdistribution.deleteTesters = loadCommand("appdistribution-testers-remove");
```

Comment:
To match the other commands, should this be `client.appdistribution.tester.add` and `client.appdistribution.testers.delete`?

Reply:
Oh wait also it should be `remove`, not `delete`